### PR TITLE
Update passt repo

### PR DESCRIPTION
**Pull request by AlonaKaplan**
**What this PR does / why we need it**:
Update the passt repo to a [fully archived mirror](https://passt.top/builds/copr/) of the Copr repository.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
This PR also includes update of the bazeldnf. 
passt rpm includes "^" which causes the generation of the WORKSPACE file to fail. The new bazeldnf version includes a [fix](https://github.com/rmohr/bazeldnf/pull/25/commits/e2e0fd8e280fe5c7820317e356808cf3425cbffb) substituting "^" with "__caret__".
The bazeldnf update commit was taken from PR #8404.

**Release note**:

```release-note
Changing the passt repo to https://passt.top/builds/copr
Updating passt version to 0^20221022.gb68da10-1.el8
```

### CODE REVIEW

File: docs/devel/networking.md [hunk 1]
```diff
@@ -271,18 +271,18 @@ take the MAC address of the first tap device attached to it,
...
 have just arrived at the network interface. This rule simply filters all
 incoming traffic from the pod networking interface, making it go through the
 `KUBEVIRT_PREINBOUND` chain.
 
 ```
-Chain PREROUTING (policy ACCEPT)
-target               prot opt in   source               destination
-KUBEVIRT_PREINBOUND  all  --  eth0 anywhere             anywhere
+chain prerouting {
+		iifname "eth0" counter jump KUBEVIRT_PREINBOUND
```
Comment (maiqueb) [inline docs/devel/networking.md:284]: please make this consistent with the other rules and use less spaces to indent it.

File: docs/devel/networking.md [hunk 2]
```diff
@@ -292,30 +292,31 @@ in the masquerade interface specification) or simply have all ports accepted -
 when the port configuration is omitted.
 
 ```
-Chain KUBEVIRT_PREINBOUND (1 references)
-target     prot opt source               destination
-DNAT       tcp  --  anywhere             anywhere             tcp dpt:http to:10.0.2.2
+chain KUBEVIRT_PREINBOUND {
+		counter dnat to 10.11.12.2
```
Comment (maiqueb) [inline docs/devel/networking.md:296]: ditto

File: docs/devel/networking.md [hunk 3]
```diff
@@ -292,30 +292,31 @@ in the masquerade interface specification) or simply have all ports accepted -
...
 the gateway -
+-
 `10.11.12.1`.
 
 ```
-Chain KUBEVIRT_POSTINBOUND (1 references)
-target     prot opt source               destination
-SNAT       tcp  --  anywhere             anywhere             tcp dpt:http to:10.0.2.1
+chain KUBEVIRT_POSTINBOUND {
+		ip saddr { 127.0.0.1 } counter snat to 10.11.12.1
```
Comment (maiqueb) [inline docs/devel/networking.md:318]: ditto

**Review by maiqueb — CHANGES_REQUESTED** [review]
This PR features something - imo - a lot more important than updating passt repo: removal of iptables packet filtering framework.

That should - imo - be moved to a different PR, highlighting it. 

I have no issue with removing it, but I was not expecting that change when checking the code diff.

Finally, I would also like to see a note in the README indicating that KubeVirt will not run in kernels <= [3.13](https://kernelnewbies.org/Linux_3.13#nftables.2C_the_successor_of_iptables).

This last point is probably not strictly required since kernel 3.13 has been available for > 8 years...

**Comment by AlonaKaplan** [comment]
> This PR features something - imo - a lot more important than updating passt repo: removal of iptables packet filtering framework.
> 
> That should - imo - be moved to a different PR, highlighting it.
> 
> I have no issue with removing it, but I was not expecting that change when checking the code diff.
> 
> Finally, I would also like to see a note in the README indicating that KubeVirt will not run in kernels <= [3.13](https://kernelnewbies.org/Linux_3.13#nftables.2C_the_successor_of_iptables).
> 
> This last point is probably not strictly required since kernel 3.13 has been available for > 8 years...

@maiqueb It is just a rebase on top of PR #8625 
Sorry for not mentioning it in the commit message.

**Comment by maiqueb** [comment]
> > This PR features something - imo - a lot more important than updating passt repo: removal of iptables packet filtering framework.
> > That should - imo - be moved to a different PR, highlighting it.
> > I have no issue with removing it, but I was not expecting that change when checking the code diff.
> > Finally, I would also like to see a note in the README indicating that KubeVirt will not run in kernels <= [3.13](https://kernelnewbies.org/Linux_3.13#nftables.2C_the_successor_of_iptables).
> > This last point is probably not strictly required since kernel 3.13 has been available for > 8 years...
> 
> @maiqueb It is just a rebase on top of PR #8625 Sorry for not mentioning it in the commit message.

OK, I see. That should be mentioned in the PR's description, with a `Depends-on: xyz>` stanza. 

Thank you for the explanation.

**Pull request closed without merging**